open_rarity/models/__init__.py

```diff
@@ -1,6 +1,6 @@
 from .collection import Collection
 from .token import Token
-from .token_identifier import EVMContractTokenIdentifier
-from .token_metadata import StringAttribute, TokenMetadata
-from .token_rarity import TokenRarity
-from .token_standard import TokenStandard
+from .token.identifier import EVMContractTokenIdentifier
```

Comment:
nit: If we're gunna update this also we should import all other identifiers, e.g.

`TokenIdentifier` and `SolanaMintAddressTokenIdentifier`